<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arrangement</class>
 <widget class="QWidget" name="arrangement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arrangement</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="availableActionsLabel">
       <property name="text">
        <string>Available actions:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="currentActionsLabel">
       <property name="text">
        <string>Current actions:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QListWidget" name="availableActionsListWidget"/>
     </item>
     <item row="2" column="2">
      <layout class="QVBoxLayout" name="currentActionsLayout">
       <item>
        <widget class="QListWidget" name="currentActionsListWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="addLauncherButton">
         <property name="text">
          <string>Add launcher</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="buttonsLayout">
       <item row="2" column="0">
        <widget class="QToolButton" name="removeButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="addButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <spacer name="centerSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="moveDownButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="moveUpButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="upSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <spacer name="downSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
